libatalk/atp/atp_open.c

```diff
@@ -50,9 +53,9 @@ ATP atp_open(uint8_t port, const struct at_addr *saddr)
     struct timeval	tv;
     int			pid;
 
-#ifdef DEBUG
+#ifdef EBUG
```

Comment:
Why do you not want to use the DEBUG label here?

Reply:
Oddly, DEBUG isn't defined in the 3.x tree. It also might have been an orphan. All the other ATP source files are using EBUG in the same places for debug messages.

Reply:
Interesting. A deep dive into git blame will probably give a hint to what happened. I agree that we should stick with the current code standard.